src/docker/10-debianconsole/.prebuild.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+# FIXME Remove this when Docker supplies multi-arch base images
+. .$ARCH.base-images
+docker pull $DEBIAN_BASE_IMAGE
```

Comment:
In general in our scripts we try not to pull if the image is already local, just to speed things up.  So you can you add a `docker inspect $DEBIAN_BASE_IMAGE >/dev/null 2>&1 || docker pull ...`